doc-turkish/integers.md

```diff
@@ -47,49 +47,49 @@ func Add(x, y int) int {
 
-If you have noticed we learnt about _named return value_ in the [last](hello-world.md#one...last...refactor?) section but aren't using the same here. It should generally be used when the meaning of the result isn't clear from context, in our case it's pretty much clear that `Add` function will add the parameters. You can refer [this](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters) wiki for more details.
+Farkettiyseniz [son](hello-world.md#one...last...refactor?) bölümde _named return value_ öğrenmiştik ama burada aynısını kullanmadık. Bu durum genelde bağlamın sonucu anlaşılır olmadığında kullanılmalıdır. Bizim durumumuzda ise `Add` fonksiyonunun parametrelerini toplayacağı oldukça açıktır. Daha fazla detay için [buraya](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters) bakabilirsiniz.
 
-## Write enough code to make it pass
+## Testi geçecek kadar kod yaz
 
-In the strictest sense of TDD we should now write the _minimal amount of code to make the test pass_. A pedantic programmer may do this
+TDD'nin tam manası ile _testi geçecek kadar minimum kod_ yazmalıyız. Belki ukala bir programcı bu şekilde yapardı.
 
 ```go
 func Add(x, y int) int {
 	return 4
 }
 ```
 
-Ah hah! Foiled again, TDD is a sham right?
+Aha! Tekrar engellendi, TDD sahtekarlık değil mi?
 
-We could write another test, with some different numbers to force that test to fail but that feels like [a game of cat and mouse](https://en.m.wikipedia.org/wiki/Cat_and_mouse).
+Testin başarısız olması için farklı numaralarla başka test yazabiliriz ancak bu [kedi fare oyununa](https://en.m.wikipedia.org/wiki/Cat_and_mouse) benzeyecek.
 
-Once we're more familiar with Go's syntax I will introduce a technique called *"Property Based Testing"*, which would stop annoying developers and help you find bugs.
+Go'nun söz dizimine aşina olduğumuzda geliştiricileri sıkmayan ve bug bulmalarına yardımcı olan _"Property Based Testing"_ isimli tekniği anlatacağım.
 
-For now, let's fix it properly
+Şimdilik düzgünce yapalım
 
 ```go
 func Add(x, y int) int {
 	return x + y
 }
 ```
 
-If you re-run the tests they should pass.
+Eğer testi çalıştırırsanız geçecektir.
 
 ## Refactor
 
-There's not a lot in the _actual_ code we can really improve on here.
+Var olan kodda iyileştirme yapabileceğimiz bir şey yok.
 
-We explored earlier how by naming the return argument it appears in the documentation but also in most developer's text editors.
+Dönüş argümanını adlandırarak bunun belgelerde ve aynı zamanda çoğu geliştiricinin editöründe nasıl göründüğünü daha önce keşfetmiştik.
 
-This is great because it aids the usability of code you are writing. It is preferable that a user can understand the usage of your code by just looking at the type signature and documentation.
+Bu harika çünkü yazdığınız kodun kullanılabilirliğine yardımcı oluyor. Kullanıcı sadece kodunuzun imzasına ve dokümanına bakarak kullanımını anlayabilir.
```

Comment:
kod imzasının ne demek olduğunu bilmiyordum, bilgisayar biliminde bir terimmiş: type signature. başkalarının da öğrenmesi için:

kodunuzun imzasına([type signature](https://en.wikipedia.org/wiki/Type_signature)) şeklinde değiştirip yönlendirilecek bir kaynak eklesek güzel olur gibi, ne dersin?